rocky/rocky/views/health.py

```diff
@@ -52,10 +52,10 @@ def get_octopoes_health(octopoes_api_connector: OctopoesAPIConnector) -> Service
     return octopoes_health
 
 
-def get_scheduler_health() -> ServiceHealth:
+def get_scheduler_health(organization_code: str) -> ServiceHealth:
     try:
-        scheduler_health = client.health()
-    except HTTPError:
+        scheduler_health = scheduler_client(organization_code).health()
+    except (SchedulerError, HTTPError):
```

Comment:
Are there still HTTPErrors expected to bubble up here? I'd suggest catching all those in the client and raising a SchedulerError (or a more specific subclass of that) to abstract away that we are dealing with a http endpoint.

Reply:
ahhh yes, I forgot to move this to the client